File: dynamicpdf-csharp-examples/Examples/TiffToPDF.cs
using ceTe.DynamicPDF;
using ceTe.DynamicPDF.Imaging;

namespace DynamicPDFCoreSuite.Examples
{
    class TiffToPDF
    {
        public static void Run()
        {
            TiffFile tiffFile = new TiffFile(Util.GetPath("Resources/Images/fw9_18.tif"));
            Document document = tiffFile.GetDocument();
            document.Draw(Util.GetPath("Output/TiffToPDF.pdf"));
        }
    }
}

File: dynamicpdf-vbnet-examples/Examples/TiffToPDF.vb
Imports ceTe.DynamicPDF
Imports ceTe.DynamicPDF.Imaging

Class TiffToPDF
    Public Shared Sub Run()
        Dim tiffFile As New TiffFile(Util.GetPath("Resources/Images/fw9_18.tif"))
        Dim document As Document = tiffFile.GetDocument()
        document.Draw(Util.GetPath("Output/TiffToPDF.pdf"))
    End Sub
End Class

